doc/src/modules/plotting.rst

```diff
@@ -78,8 +78,36 @@ Series Classes
 .. autoclass:: sympy.plotting.plot_implicit.ImplicitSeries
    :members:
 
-
-
+Examples
+========
+
+Below are examples of 2D plots.
+
+Example 1
+---------
+
+**append(** *arg* **)**
+
+Adds an element from a plot’s series to an existing plot.
+Consider two Plot objects, p1 and p2. To add the second plot’s first series object to the first, use the append method, like so:
+
+.. plot::
+   :context:
+   :format: doctest
+   :include-source: True
+   
+   >>> from sympy import symbols
+   >>> from sympy.plotting import plot
+   >>> x = symbols('x')
+   >>> p1 = plot(x*x)
+   >>> p2 = plot(x)
+   >>> p1.append(p2[0])
+   >>> p1
+   Plot object containing:
+   [0]: cartesian line: x**2 for x over (-10.0, 10.0)
+   [1]: cartesian line: x for x over (-10.0, 10.0)
+
+.. image:: ../pics/append.png
```

Comment:
This is not needed, it is auto generated by the extension. Build the docs locally and make sure it works, then paste a screenshot here.

Reply:
![screenshot](https://user-images.githubusercontent.com/35144226/46459265-03792580-c7d6-11e8-8f92-41074ab48663.png)
I ran the sphinx script on my local machine and got this kind of output.

Reply:
Ok, so from what I've understood, I'm supposed to add these changes in the sympy plotting docs (https://github.com/sympy/sympy/blob/master/doc/src/modules/plotting.rst) instead of building my own docs. But that is what I did when I opened the pull request. Should I just add the example as I did before, but run it on my machine and perform the doctest? I'm kind of confused about this:
> This is not needed, it is auto generated by the extension. Build the docs locally and make sure it works, then paste a screenshot here.

Reply:
The example you are using is from here: https://github.com/sympy/sympy/blob/master/sympy/plotting/plot.py#L230 

You should modify that file, not create a new duplicate example.

Reply:
Ok, so I'll just add the output in that file and then post a screenshot here.

Reply:
Ok, so finally, I got this:
![ss1](https://user-images.githubusercontent.com/35144226/46499564-0f98cd80-c83e-11e8-880d-73b8eded4663.png)
And these are the doctest results
![ss2_tests](https://user-images.githubusercontent.com/35144226/46499587-1fb0ad00-c83e-11e8-99ba-885203033786.png)

Reply:
That's it!

Reply:
Commit your changes and push to this branch.